tests/nft/nft_wash_trades_assert_unique.sql

```diff
@@ -0,0 +1,8 @@
+-- Check for multiple unique_trade_id
+
+SELECT unique_trade_id
```

Comment:
this test makes more sense as you are filtering for last two days. we could also build a generic test which is a dbt macro that accepts table name, column name, number of days to filter back. note that i already have a unique_filtered generic macro under generic tests, you could apply similarly to my other comment, but use - unique_filtered instead (assuming a week is okay)

Reply:
https://github.com/duneanalytics/spellbook/blob/main/tests/generic/is_unique_filtered.sql

Reply:
we can edit that macro to be hardcoded to last two days vs. the select max(), as there have been issues with that approach. i don't believe this macro is used much, so not a harm to edit